test/primitives/array-element-test.js

```diff
@@ -123,6 +123,16 @@ describe('ArrayElement', function() {
       });
     });
 
+    describe('#getIn', function() {
+      var arr = new minim.ArrayElement([
+        {
+          a: [1, 2, 3]
+        }
+      ]);
+
+      expect(arr.getIn([0, 'a', 1]).toValue()).to.equal(2);
```

Comment:
This test will not work, as equality will not be met since they are different instances of and object. It's like when you say:

```js
expect({ foo: 'bar'}).to.equal({ foo: 'bar'})
```

and you get a failing test. You have to use deep equal for that situation to test against the actual values given.

So I'm not actually sure the of the way to do this without converting back to a value and comparing against primitive values. I could compare against the content of the value, but I am trying to stay away from accessing content directly. I'm open to other ideas, but I'm afraid `.toValue